### ISSUE TITLE
system suspend during an active voice calll

### ISSUE THREAD

**Issue by AndreySV** [telegramdesktop/tdesktop#26008]
### Steps to reproduce

1. Use battery powered device (laptop) or any other system, that is configured to automatically suspend after some time of inactivity.
2. Make a voice call
3. Talk long enough to trigger system suspend

### Expected behaviour

system is not suspended, talk continues

### Actual behaviour

system suspends, talk aborts

### Operating system

Debian 11/GNOME 3.38, Debian 12/GNOME 43.

### Version of Telegram Desktop

4.6.5 (official build, that updates itself)

### Installation source

Other (unofficial) source

### Logs

```shell
Usefull commands for testing under GNOME set 1.5 minute to system suspend:

gsettings set org.gnome.settings-daemon.plugins.power sleep-inactive-battery-timeout 90
gsettings set org.gnome.settings-daemon.plugins.power sleep-ac-battery-timeout 90
```
There are systemd-inhibit or gnome-session-inhibit that could be used to temporary block system suspend.
```

**Comment by Aokromes** [comment]
sort of duplicate of https://github.com/telegramdesktop/tdesktop/issues/7790

**Comment by ilya-fedin** [comment]
#7790 was fixed long time ago, in 3.5.1: https://github.com/telegramdesktop/tdesktop/commit/9c01295521c7f93f9496c851d0304ce9f0db0112. There seem to be actual bug due to missing suspend inhibiton on Linux, the code was written, but was disabled since https://github.com/desktop-app/lib_base/commit/1ec9a0635231a37e18fbe6291a58c685cdf75d5b due to #24032, #24043, https://bugs.telegram.org/c/14426. So there's only support for display sleep prevention (but support is missing on GNOME end, see https://gitlab.gnome.org/GNOME/mutter/-/merge_requests/111, switching to X11 session should help), no PC sleep prevention. Calls are using display sleep prevention,so this exact issue is a GNOME bug.

**Comment by ilya-fedin** [comment]
PC sleep inhibiton is blocked on https://github.com/flatpak/xdg-desktop-portal/issues/707, until portal is fixed, we can't use its API, at least in this decade due to backward compatibility.
